radar-auth/src/main/java/org/radarcns/auth/config/YamlServerConfig.java

```diff
@@ -40,39 +38,44 @@
      * @return The initialized configuration object based on the contents of the configuration file
      * @throws ConfigurationException If there is any problem loading the configuration
      */
-    public static YamlServerConfig readFromFileOrClasspath() {
+    public YamlServerConfig loadConfig() {
```

Comment:
Why is this? It makes sense for it to be a static function. Perhaps the stored `config` variable could be removed though, and it can always be loaded from scratch. Alternatively, it could use a singleton pattern, but just loading it every time makes more sense to me.

Reply:
Codacy suggested to remove static singletons. or it should be synchronized. I will have a look again

Reply:
Indeed, and if the `config` variable is removed it doesn't have to be synchronised, then it is completely stateless.

Reply:
That would make the file to be loaded every time we call the method. Is that what we expect?